tests/test_warp_transform.py

```diff
@@ -173,3 +174,9 @@ def test_transform_bounds_identity():
     """Confirm fix of #1411"""
     bounds = (12978395.906596646, 146759.09430753812, 12983287.876406897, 151651.06411778927)
     assert transform_bounds("+init=epsg:3857", "+init=epsg:3857", *bounds) == bounds
+
+
+def test_issue1131():
+    """Confirm that we don't run out of memory"""
+    transform, w, h = calculate_default_transform(CRS.from_epsg(4326), CRS.from_epsg(3857), 455880, 454450, 13.0460235139, 42.6925552354, 13.2511695428, 42.8970561511)
+    assert (w, h) == (381595, 518398)
```

Comment:
These extents are 100x the virtual volume of pixels that caused the memory error reported in #1131, so I think we're in good shape.